aspnet/fundamentals/logging/sample/src/LoggingSample/SimpleLoggerMiddleware.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.Logging;
+using System.Threading.Tasks;
+
+namespace LoggingSample
+{
+    public class SimpleLoggerMiddleware
+    {
+        private readonly RequestDelegate _next;
+        private readonly ILogger<SimpleLoggerMiddleware> _logger;
+
+        public SimpleLoggerMiddleware(RequestDelegate next, 
+            ILogger<SimpleLoggerMiddleware> logger)
+        {
+            _next = next;
+            _logger = logger;
+        }
+
+        public async Task Invoke(HttpContext context)
+        {
+            _logger.LogInformation("Handling request: {path}", 
+                context.Request.Path);
+
+            await _next.Invoke(context);
+
+            _logger.LogInformation("Finished handling request.");
```

Comment:
This is pretty much a consequence of me trying to keep the sample as small and tight as possible, without having to pull in MVC for instance. Based on this and above feedback I think I need to boost the sample to something more real-world than tiny middleware wrappers. I'll come up with a minimal MVC application that can work for this purpose and add some logging to specific user actions within the app.